s3/src/main/java/com/sap/pls/samples/s3/Controller.java

```diff
@@ -119,6 +120,10 @@ public ResponseEntity listAccessPoints(@PathVariable("bucketName") String bucket
 
     @GetMapping(value = "/accesspoint/{fileName}", produces = "application/json; charset=UTF-8")
     public ResponseEntity getObjectAccessPoint(@PathVariable("fileName") String fileName) {
+        if (!StringUtils.hasText(this.config.getAccessPointArn())) {
```

Comment:
Shall we just check if the string is empty? 

```suggestion
        if (this.config.getAccessPointArn().isEmpty()) {
```

Reply:
We will need to check if it's null since it's noted as an optional value in the readme and `getString` returns `null` in case the label is not found.

See our code: 

https://github.com/SAP-samples/private-link-aws-services/blob/644a4e1ee45b5a6639661cbec91c5ade05a75a4d/s3/src/main/java/com/sap/pls/samples/s3/Application.java#L91

And the `java-cfenv` lib:

https://github.com/pivotal-cf/java-cfenv/blob/48f2ab5d49fef664e562574a90898e65ade6d047/java-cfenv/src/main/java/io/pivotal/cfenv/core/CfCredentials.java#L173-L181